proxy: Ignore visibility annotation if proxy is disabled (#27597)
cilium-team/cilium: Documentation/observability/visibility.rst

Layer 7 Protocol Visibility
***************************

.. note::

>     This feature requires L7 Proxy support to be enabled. If not, the visibility annotation will be ignored.

Reviewer: Use present tense, use active voice
```suggestion
    This feature requires enabling L7 Proxy support. Without it, the visibility annotation is ignored.
```

Author: Thanks for your input :+1:
